### DISCUSSION TITLE
Move EDC f&q to tractusx-edc/discussions

### DISCUSSION THREAD

**Discussion by stephanbcbauer** [eclipse-tractusx/tractusx-edc#185] [Ideas]
**current situation**

Right now we have a lot of discussions on teams. The "generated" answers could be useful for everybody. Since the teams chat is restricted... and deletes messages older than x ( I think three month) we are losing information

**proposal**

Move the discussion to tractusx-edc/discussion.

**Comment by paullatzelsperger** [comment]
Another option, maybe as an auxiliary measure, would be to collect questions in a FAQ section. OpenSource projects often have "Office Hours" during which users can ask questions, which should then be collected in that FAQ section.

**Reply by stephanbcbauer** [reply]
I think that's good, but when I look at the team chat, the topics always seem to be rather acute :)

**Comment by stephanbcbauer** [comment]
I think this discussion can be closed, since we are moving step by step to discussions. Also, the “Office hour” is implemented with the extended weekly → both fine for me.
